CMSgov/design-system
# [WNMGDS-1205] help drawer toggle icon replacement

## Summary
Replacing the info icon in the mgov HelpDrawerToggle component with an inline svg instead of a CSS background image

## How to test
Run `yarn build` to build core. Run `yarn build:medicare` to build medicare and get updated core code. Run `yarn storybook:medicare` and you should see the new HelpDrawer story in storybook

Outstanding questions:
* Can we get a new icon with separate elements for easier CSS styling?
* Can this CSS workaround be a temporary solution?
* How do we deal with the inverse version of this button? (When the button is on a dark background, can we set a color to be the outline & "i" color that will work for all cases)
